app.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import 'highcharts';
+import ReactHighcharts from 'react-highcharts';
+
+/**
+ * Component representing a single char
+ */
+class Chart extends React.Component {
+    /**
+     * Create a new chart
+     * @param props props of the chart
+     */
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: null,
+            error: null
+        };
+    }
+
+    /**
+     * @inheritdoc
+     */
+    componentDidMount() {
+        fetch('./psnrResults')
+            .then(res => {
+                if (res.status === 200) {
+                    res.json().then(jsonData => {
+                        console.log(jsonData);
+                        this.setState({
+                            data: jsonData.map(jd => [ jd.buildNum, jd.psnr ])
+                        });
+                    });
+                } else {
+                    this.setState({
+                        error: 'Error retrieving psnr results'
+                    });
+                }
+            })
+            .catch(err => {
+                this.setState({
+                    error: err
+                });
+            });
+    }
+
+    /**
+     * @inheritdoc
+     */
+    render() {
+        if (this.state.error) {
+            return (
+                <div>Error: {this.state.error}</div>
+            );
+        }
+        if (this.state.data) {
+            const config = {
+                title: {
+                    text: 'PSNR'
+                },
+                yAxis: {
+                    title: {
+                        text: 'PSNR value'
+                    }
+                },
+                xAxis: {
+                    title: {
+                        text: 'Build number'
+                    }
+                },
+                series: [
+                    {
+                        data: this.state.data
+                    }
+                ]
+            };
+
+            return (
+                <ReactHighcharts config = {config}></ReactHighcharts>
+            );
+        }
+
+        return (
+            <div>Loading...</div>
+        );
+    }
+}
+
+/**
+ * Component for displaying the top-level dashboard
+ */
+class Dashboard extends React.Component {
+    /**
+     * @inheritdoc
+     */
+    render() {
+        return (
+            <Chart />
+        );
+    }
+}
+
+export default Dashboard;
```

Comment:
I think there are some things in here that could be changed but I'm also thinking it's fine for a first commit. Some things to change might be to be explicit about loading state, separating components into separate files, returning res.json() from the promise, creating a wrapper around fetch to hide that impl, and others... but they're all things that aren't needed right now, and honestly probably shouldn't be done right now, but feel I should just at least mention.